rss2email/feeds.py

```diff
@@ -43,7 +43,11 @@
 from . import error as _error
 from . import feed as _feed
 
-import fcntl as _fcntl
+try:
+    import fcntl as _fcntl
+    UNIX = True
+except ImportError:
```

Comment:
Sorry about breaking this for Windows in the past. I suppose the correct thing for me to do would be to have a cross-platform lock object here, so we don't have to do this import check.

Reply:
That’s OK, it happens =).